.circleci/config.yml

```diff
@@ -366,11 +366,31 @@ jobs:
       - gcp-gcr/push-image:
           image: bigquery-etl
           tag: ${CIRCLE_TAG:-latest}
+  fail-on-fork:
+    docker: *docker
+    steps:
+      - checkout
+      - run:
+          name: Manually trigger integration tests for fork
+          command: |
+            echo "Integration tests for this fork need to be triggered manually"
+            echo "Users with write access to the repository can trigger" \
+              " integration tests by visiting: "
+            echo "https://github.com/mozilla/bigquery-etl/actions/workflows" \
+              "/push-to-upstream.yml".
+            echo "Trigger via 'Run workflow' and provide the " \
+              "<username>:<branch>" of this PR as parameters."
```

Comment:
Do we have sufficient info in env vars here that we can inject the actual values into this message?

Reply:
Ah yes that is possible! Updated the task and included the branch label in the output